### PR TITLE
ACMS-1867: Release PHP 8.2 compatible ACMS.

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #[ACMS-1867](https://backlog.acquia.com/browse/ACMS-1867)

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: .github/workflows/acquia_cms_ci.workflow.yml [hunk 1]
```diff
@@ -159,6 +159,12 @@ jobs:
 #          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
       - name: Before Install
         run: |
+          # Update the CI by adding patches without pinning the following modules.
+          composer require "drupal/facets:^2.0.6" --no-update --no-install -d modules/acquia_cms_search
+          composer require "drupal/pathauto:^1.11" --no-update --no-install -d modules/acquia_cms_common
+          composer require "drupal/redirect:^1.8" --no-update --no-install -d modules/acquia_cms_common
+          composer require "drupal/jsonapi_extras:^3.23" --no-update --no-install -d modules/acquia_cms_headless
```
Comment (vishalkhode1) [inline .github/workflows/acquia_cms_ci.workflow.yml:166]:
```suggestion
          composer require "drupal/jsonapi_extras:^3.23" --no-update --no-install -d modules/acquia_cms_headless
          composer require "drupal/jsonapi_extras:^3.23" --no-update --no-install -d modules/acquia_cms_component          
```

File: modules/acquia_cms_headless/composer.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "license": "GPL-2.0-or-later",
     "require": {
         "drupal/acquia_cms_tour": "2.x-dev",
-        "drupal/jsonapi_extras": "^3.23",
+        "drupal/jsonapi_extras": "3.23",
```
Comment (vishalkhode1) [inline modules/acquia_cms_headless/composer.json:8]: @chandan-singh7929  This needs to be added for acquia_cms_component module as well.

**Review by vishalkhode1 — CHANGES_REQUESTED** [review]

File: modules/acquia_cms_headless/composer.json (hunk 2 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_headless/composer.json:8]: Thanks @vishalkhode1 I have updated it.

File: .github/workflows/acquia_cms_ci.workflow.yml (hunk 1 above)
Comment (chandan-singh7929) [reply .github/workflows/acquia_cms_ci.workflow.yml:166]: Done.

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**